assets/js/stripe-payment-request.js

```diff
@@ -406,12 +406,9 @@ jQuery( function( $ ) {
 					}
 
 					if ( $( '#wc-stripe-payment-request-button' ).length ) {
+						$( '#wc-stripe-payment-request-wrapper, #wc-stripe-payment-request-button-separator' ).show();
```

Comment:
Why did you move the `show()` call to before the `mount()` call?

Reply:
Not sure if it makes a difference but I'd felt there was a slim chance that the mounting of the button could suffer from the container being hidden (if it reads any layout measurements, for instance), and it made sense to me that the separator would also just be shown earlier as well. Didn't actually test it the other way, but that was why – do you feel differently / prefer to keep one or both elements hidden until after mounting?

Reply:
I was just curious if this change was intentional, since before this `mount()` was before `show()` and seemed to be working well. Your explanation makes sense, and worst-case-scenario, it just makes no difference :)